File: lib/otel_sample/otel_sample_child_workflow.ex
defmodule OtelSampleChild.Workflow do
  use TemporalSdk.Workflow
  @moduledoc false

  @impl true
  def execute(context, _input) do
    %{workflow_info: %{opentelemetry_baggage: _baggage}} = context
    :ok
  end
end

File: src/otel_sample/otel_sample_child_workflow.erl
-module(otel_sample_child_workflow).

% elp:ignore W0012 W0040
-moduledoc false.

-export([
    execute/2
]).

-include_lib("temporal_sdk/include/workflow.hrl").

execute(Context, _Input) ->
    #{workflow_info := #{opentelemetry_baggage := _Baggage}} = Context,
    ok.
